[2025-06-27T06:23:47.614+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:23:47.638+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.end_task scheduled__2025-02-24T02:10:00+00:00 [queued]>
[2025-06-27T06:23:47.650+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.end_task scheduled__2025-02-24T02:10:00+00:00 [queued]>
[2025-06-27T06:23:47.653+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:23:47.700+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): end_task> on 2025-02-24 02:10:00+00:00
[2025-06-27T06:23:47.728+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1294) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:23:47.736+0000] {standard_task_runner.py:72} INFO - Started process 1327 to run task
[2025-06-27T06:23:47.743+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_dev', 'end_task', 'scheduled__2025-02-24T02:10:00+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpgeed7lus']
[2025-06-27T06:23:47.754+0000] {standard_task_runner.py:105} INFO - Job 319: Subtask end_task
[2025-06-27T06:23:47.904+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_dev.end_task scheduled__2025-02-24T02:10:00+00:00 [running]> on host 4e2c61145aac
[2025-06-27T06:23:48.036+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_dev' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-24T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-24T02:10:00+00:00'
[2025-06-27T06:23:48.045+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:23:48.057+0000] {logging_mixin.py:190} INFO - end
[2025-06-27T06:23:48.058+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T06:23:48.078+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:23:48.078+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_dev, task_id=end_task, run_id=scheduled__2025-02-24T02:10:00+00:00, execution_date=20250224T021000, start_date=20250627T062347, end_date=20250627T062348
[2025-06-27T06:23:48.139+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:23:48.165+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
